### DISCUSSION TITLE
How can one override `doom-<module>.lua` configs from `doom_userpluggins.lua`

### DISCUSSION THREAD

**Discussion by molleweide** [NTBBloodbath/doom-nvim#267] [Q&A] [answered]
Hey guys,

Is it possible to override doom module configurations from `doom_config.lua` or `doom_userplugins.lua` somehow?

**Comment by connorgmeehan** [comment, marked as answer]
Apart from the options provided in doom_config.lua there's not a way to override the config right now.  What are you trying to do?  And do you have an idea of how you'd like to override the default config or what that might look like as a user?

**Comment by molleweide** [comment]
first of all it was interesting to just see your reply because it makes sense.
mostly there are some keybindings that I have changed in core for completion, kommentary, and luasnip, and now that I wanted to make my config free of messing around in core for main I was wondering if this was possible to do but I know that you guys are working on new architecture and stuff so this is not too big of a deal now.

**Reply by molleweide** [reply]
so that I can start with the docker so that I can continuosly transfer my stuff into develop.
Now I have pretty muched moved everything into main and it works!! so now i am happy.

**Reply by molleweide** [reply]
I have created a lot of search and replace commands that I would like to move into core but I failed when trying to create a new mappings file under extra/keybindings/search-replace.lua. so I want to see how you guys would add the file so that I can show you my bindings. or you could also just look under my `doom_config` file and pick the commands because some of them are really good atm and some of them need some working.

when I have been looking into you guys configs I haven't seen any search and replace bindings. 

maybe I should create an issue for this!
